openscm/timeseries_converter.py

```diff
@@ -235,30 +233,53 @@ def _calc_continuous_representation(
             interpolated timeseries at that point in time ("y-value").
         """
         if self._timeseries_type == ParameterType.AVERAGE_TIMESERIES:
-            if self._interpolation_type == InterpolationType.LINEAR:
```

Comment:
So we want to stick to these interpolation points regardless of the order of interpolation used?

Reply:
oops removing that was an errror good pick up!